sw/airborne/arch/chibios/mcu_arch.h

```diff
@@ -44,4 +44,40 @@ extern void mcu_arch_init(void);
 extern bool recovering_from_hard_fault;
 #endif
 
+#include <ch.h>
+
+static inline void mcu_deep_sleep(void)
```

Comment:
doxygen comment for this function explaining the need and difference to a normal sleep would be nice